package.json

```diff
@@ -51,7 +51,6 @@
     "@typescript-eslint/eslint-plugin": "^5.30.5",
     "@typescript-eslint/parser": "^5.30.5",
     "chai": "^4.3.7",
-    "dotenv": "^16.0.1",
```

Comment:
why did we remove `dotenv`?

Reply:
I didn't remove it, it was duplicated on devdep

Reply:
Got it